app/sequential.py

```diff
@@ -6,5 +6,5 @@ def __init__(self, operation, planning_svc):
 
     async def execute(self, phase):
         for member in self.operation.agents:
```

Comment:
With the change mentioned in the other comment on the main PR (Agent is optional) you can remove this loop and just have the `get_links` provide every possible link for all agents